README.md

```diff
@@ -0,0 +1,341 @@
+Working with EWP Technical Documentation
+========================================
+
+This document is our proposal on **how** to work with the technical
+documentation within the EWP project (where to store it, how to submit change
+proposals, etc.).
+
+* [What is the status of this document?][statuses]
+* [See the index of all other EWP Specifications][develhub]
+
+
+The Structure of EWP Repositories
+---------------------------------
+
+### How documentation is to be divided?
+
+ * Separate sections of the documentation SHOULD be stored in **separate
+   Git repositories**.
+
+ * The index of all documents (along with their current statuses) SHOULD be
+   kept [at this URL][develhub]. If SHOULD be frequently updated.
+
+Some of these repositories will probably not change much after their first
+release, while some others (like the APIs) will probably keep mutating perhaps
+even after the project is finished (though, in a backward-compatible way).
+
+
+### Why separate API repositories?
+
+We chose to split EWP requirements into a couple of separate APIs, and host
+them in separate GitHub repositories:
+
+ * We believe that partners (this includes all the **future** partners) should
+   have the option to implement just a **subset** of all the features we have
+   documented. This will make it easier for them to start off, claim that they
+   have parts of EWP properly implemented, and then implement other parts later
+   (possibly *much* later, with other teams of developers).
+
+ * Having each API in **separate repository** has some great advantages, when
+   it comes to versioning:
+
+   - If a partner does not implement *API X*, then he also knows that he may
+     ignore all the changes in *API X*'s repository.
+
+   - If a partner has already implemented *API Y* and wants to upgrade his
+     implementation to include some new changes in new API release, he may
+     simply review a diff (and/or release notes) in the Git repository between
+     specific tags.
+
+   - It allows us to use [semantic versioning](http://semver.org/) separately
+     for each API. This allows the partners to easily spot
+     backward-incompatible changes which may break their implementations.
+
+ * It allows the project leaders to choose and manage separate teams for
+   working with separate APIs.
+
+
+### Why GitHub?
+
+We chose to use Git and GitHub as the primary tool for working and hosting
+*documentation for developers*:
+
+ * All developers are familiar with Git. Many developers are also familiar with
+   GitHub. **We will require developers to review and accept documentation
+   often** throughout the project. We believe that these tools will help them
+   with this task.
+
```

Comment:
I'd say "All developers are familiar with Git" is a bit too far fetched, there are other version control systems, and some people have been using them before Git existed and may even find the transition uncomfortable or not have the time to learn new "tricks"